Chinese/轮回

```diff
@@ -0,0 +1 @@
+醒来之后，发现毫无记忆，但是面前却有一书架的书，打开其中一本发现是一本日记，第一页写着作者的名字和年代：婉儿--1980-2000.打开另外一本，同样也有名字和年代：婉儿 1900-1920。似乎这个婉儿一直从1900年活到2000年，很是长寿。打开书架第一层的一本：婉儿 公元20-公元40年。手上的书滑落，这一切太过惊悚了吧。这个叫婉儿的居然活了2000年。
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.